Documentation/resources.adoc

```diff
@@ -74,6 +74,43 @@ Expose the user-defined Alertmanager web server within the cluster on the follow
 
 Expose the Alertmanager web server within the cluster on the following ports:
 * Port 9094 provides access to all the Alertmanager endpoints. Granting access requires binding a user to the `monitoring-alertmanager-view` role (for read-only operations) or `monitoring-alertmanager-edit` role in the `openshift-monitoring` project.
+----
+# monitoring-alertmanager-view grants read permissions.
+$ oc project openshift-monitoring
```

Comment:
In the docs it is better to remove comments from the code snippet. So what about something like this:
(and I also added `[source,terminal]`)

```suggestion
.List of commands you can run with the `monitoring-alertmanager-view` role (read permissions)
[source,terminal]
----
# monitoring-alertmanager-view grants read permissions.
$ oc project openshift-monitoring
```

Reply:
Also, if we want the commands to be indented correctly under the Port 9094 bullet point, we need to add a plus sign:
```
+
.List of commands you can run...
```